src/theory/rewriter.cpp

```diff
@@ -386,7 +386,8 @@ Node Rewriter::rewriteTo(theory::TheoryId theoryId,
                                  cached,
                                  PfRule::TRUST_REWRITE,
                                  {},
-                                 {eq});
+                                 {eq},
+                                 false);
```

Comment:
that last argument is the default. Moreover, also maybe add some documentation to say why this is a post-rewrite? Does not seem so clear to me.

Reply:
Will add